docs/experiments-disk12.md

```diff
@@ -5,11 +5,10 @@
 Typical indexing command:
 
 ```
-nohup sh target/appassembler/bin/IndexCollection -collection TrecCollection \
- -input /path/to/disk12/ -generator JsoupGenerator \
- -index lucene-index.disk12.pos+docvectors+rawdocs -threads 16 \
- -storePositions -storeDocvectors -storeRawDocs -optimize \
- >& log.disk12.pos+docvectors+rawdocs &
+nohup target/appassembler/bin/IndexCollection -collection TrecCollection \
```

Comment:
revert indexing command?

Reply:
Oh, I see your indexing command is auto-gen'ed.
Did you want to keep the actual tuna path?